usaspending_api/submissions/management/commands/rm_submission.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime
+import logging
+import os
+from django.core.management.base import BaseCommand
+
+from usaspending_api.submissions.models import SubmissionAttributes
+
+
+class Command(BaseCommand):
+    """
+    This command will remove a submission and all associated data with it from the
+    database
+    """
+    help = "Removes a single submission from the configured data broker database"
+    logger = logging.getLogger('console')
+
+    def add_arguments(self, parser):
+        parser.add_argument('submission_id', nargs=1, help='the broker submission id to delete', type=int)
+
+    def handle(self, *args, **options):
+        # This will throw an exception and exit the command if the id doesn't exist
+        submission = SubmissionAttributes.objects.get(broker_submission_id=options["submission_id"][0])
```

Comment:
On a nonexistent submission number, this throws an error stack ending in 

`usaspending_api.submissions.models.DoesNotExist: SubmissionAttributes matching query does not exist.`

which is not too bad as cryptic errors go, especially for a specialty task like deleting a submission, we just may want to add a better error message at some point (not necessarily part of this PR).